Sources/TuistLoader/Utils/ManifestFilesLocator.swift

```diff
@@ -23,18 +31,42 @@ public final class ManifestFilesLocator: ManifestFilesLocating {
         self.rootDirectoryLocator = rootDirectoryLocator
     }
 
-    public func locate(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
+    public func locateProjectManifests(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
         Manifest.allCases.compactMap { manifest in
             let path = at.appending(component: manifest.fileName(at))
             if FileHandler.shared.exists(path) { return (manifest, path) }
             return nil
         }
     }
 
-    public func locateAll(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
-        guard let rootPath = rootDirectoryLocator.locate(from: at) else { return locate(at: at) }
+    public func locateAllProjectManifests(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
+        guard let rootPath = rootDirectoryLocator.locate(from: at) else { return locateProjectManifests(at: at) }
         let projectsPaths = FileHandler.shared.glob(rootPath, glob: "**/\(Manifest.project.fileName(at))").map { (Manifest.project, $0) }
         let workspacesPaths = FileHandler.shared.glob(rootPath, glob: "**/\(Manifest.workspace.fileName(at))").map { (Manifest.workspace, $0) }
         return projectsPaths + workspacesPaths
     }
+
+    public func locateConfig(at: AbsolutePath) -> AbsolutePath? {
+        let subPath = RelativePath("\(Constants.tuistDirectoryName)/\(Manifest.config.fileName(at))")
+        return traverseAndLocate(at: at, appending: subPath)
+    }
+
+    public func locateSetup(at: AbsolutePath) -> AbsolutePath? {
```

Comment:
I'd add a unit test that covers this method.

Reply:
There are tests for this method. Check: `Tests/TuistLoaderTests/Utils/ManifestFileLocatorTests.swift`